### PR TITLE
feat: add headline component

### PR DESCRIPTION
**Pull request by danclark18**

### CODE REVIEW

File: packages/headline/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# npm
+$ npm add @times-visuals/headline
+```
+
+## Usage
+
+```jsx
+import Headline from "@times-visuals/headline";
+
+export default () => <Headline headline="This is a headline" />;
```
Comment (chrishutchinson) [inline packages/headline/README.md]: I think this might be a better API:

```suggestion
export default () => <Headline>This is a headline</Headline>;
```

Comment (danclark18) [reply packages/headline/README.md]: Would this mean we'd have three separate packages for headline/standfirst/label?

Comment (chrishutchinson) [reply packages/headline/README.md]: Ah no. Keep the other props the same, so you're full API would be:

```jsx
<Headline label="My label" standfirst="My standfirst">Headline goes here</Headline>
```

Comment (danclark18) [reply packages/headline/README.md]: Ah ok. If I do that how do I access the 'Headline goes here' text in the component itself?

Comment (chrishutchinson) [reply packages/headline/README.md]: Have a look at this, hopefully it'll help: https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891

If not, let me know and I can point you in the right direction

Comment (danclark18) [reply packages/headline/README.md]: Ah thank you. Think I've worked it out and pushed my changes

**Pull request merged**